ui/src/components/config_section/index.scss

```diff
@@ -10,6 +10,9 @@
 
 .euiPanel--configSection {
   .euiDescriptionList {
+    column-gap: 0px !important;
+    row-gap: 0px !important;
```

Comment:
These new styles are added in Merlin too https://github.com/caraml-dev/merlin/pull/591/files#diff-14723dd04167712f159c67e66e037f4b9e4607f247cba2ef67e474bbf45e423bR29 because the description list component now has gaps by default between each cell.